deptool/src/main/java/io/jmix/dependency/cli/command/ExportCommand.java

```diff
@@ -66,6 +86,22 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
                     return FileVisitResult.CONTINUE;
                 }
             });
+
+            if (!exportedDependencies.isEmpty()) {
```

Comment:
I would move this block out of the try-catch block. The current try-catch block is for copying dependencies, not for writing the report file.

Reply:
Will be fixed